docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p># For clarity</p><p>_, accuracy = model.evaluate()</p><p>return accuracy</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Updated